sklearn/linear_model/tests/test_ridge.py

```diff
@@ -233,6 +233,8 @@ def test_ridge_regression_hstacked_X(
     coef = coef[:-1]
 
     assert model.intercept_ == pytest.approx(intercept)
+    # coefficients are not all on the same magnitude, adding a small atol to
+    # make this test less brittle
     assert_allclose(model.coef_, np.r_[coef, coef])
```

Comment:
```suggestion
    assert_allclose(model.coef_, np.r_[coef, coef], atol=1e-8)
```

Reply:
I missed that you have removed the atol. I still think it's safer because for some seeds the magnitude of the coefs are not all the same.

Reply:
One or the other :smirk:

Reply:
or both :)